### ISSUE TITLE
optional chaining  is breaking cml in cml images

### ISSUE THREAD

**Issue by DavidGOrtega** [iterative/cml#951]
```sh
/usr/lib/node_modules/@dvcorg/cml/src/drivers/bitbucket_cloud.js:[38](https://github.com/DavidGOrtega/fashion_mnist/runs/5992622028?check_suite_focus=true#step:6:38)3
        responseBody?.error?.message || responseBody?.error || responseBody;
                     ^

SyntaxError: Unexpected token '.'
    at wrapSafe (internal/modules/cjs/loader.js:915:16)
    at Module._compile (internal/modules/cjs/loader.js:963:27)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
    at Module.load (internal/modules/cjs/loader.js:863:32)
    at Function.Module._load (internal/modules/cjs/loader.js:708:14)
    at Module.require (internal/modules/cjs/loader.js:887:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (/usr/lib/node_modules/@dvcorg/cml/src/cml.js:11:24)
    at Module._compile (internal/modules/cjs/loader.js:999:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
```

**Comment by DavidGOrtega** [comment]
@0x2b3bfa0 I had bad feeling about optional chaining

**Comment by 0x2b3bfa0** [comment]
> [_Enables Optional Chaining by default_](https://nodejs.org/gl/blog/release/v14.0.0/) — Node.js 14.0.0

**Comment by 0x2b3bfa0** [comment]
> [_In the latest update to our GitHub-hosted runners virtual environments, Node.js 16 has become the default version of `node`_](https://github.blog/changelog/2021-12-10-github-actions-github-hosted-runners-now-run-node-js-16-by-default/)

**Comment by DavidGOrtega** [comment]
True. The error comes from our images

**Comment by DavidGOrtega** [comment]
The error is in azure. Images are not the guilty @0x2b3bfa0

**Comment by 0x2b3bfa0** [comment]
(And now I reopen the issue again until we release TPI) 😄

**Comment by 0x2b3bfa0** [comment]
Closed[^1] with [v0.10.5](https://github.com/iterative/terraform-provider-iterative/releases/tag/v0.10.5)

[^1]: I've never had such a fun time like now closing and reopening issues several times. 😉
